src/providers/buildBrandColorVariants.js

```diff
@@ -0,0 +1,66 @@
+import { hslToHex, hexToHsl } from '../utils'
+
+const changeLightnessToHsl = ([h, s, l], percentage) => [h, s, l + percentage]
+
+const changePerceivedLightnessToHex = (hex, percentage) => {
```

Comment:
Maybe it would make sense to move this in `utils` next to all other color transforming utilities?